### PR TITLE
PHP 8: Code Review `Help`

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Hi @alex40724,

I completed the review of the `Service/Help` component.

Results:
- [x] The code style is `PSR-2 + X` compliant: 1 File has been changed by the `PHP-CS-FIXER`
- [x] No usage of `$_GET` / `$_POST` / `$_REQUEST` / `$_SESSION`
- [x] There is a `Unit Test Suite` with at least one unit test
- [x] The unit tests pass
- [x] External libraries are compatible with PHP 8 (if relevant)
- [x] There are no serious `PhpStorm Code Inspection` issues left
- [ ] The types are fully documented (PhpDoc) or explict PHP types are used (type hints, return types, typed properties)

I also fixed some trivial issues reported by my inspection profile.

--

Actions needed:

- [ ] Please check my inline comments. Some of them are just information, some require an action. You can use `TODO PHP8-REVIEW` when searching.

Best regards,
@mjansenDatabay

### CODE REVIEW

File: Services/Help/classes/class.ilObjHelpSettings.php [hunk 1]
```diff
@@ -71,13 +71,13 @@ public static function writeHelpModuleLmId(
...
     ) : void {
-        $id = $this->createHelpModule();
+        $id = self::createHelpModule();
         
         try {
             $imp = new ilImport();
             $conf = $imp->getConfig("Services/Help");
             $conf->setModuleId($id);
-            $new_id = $imp->importObject("", $a_file["tmp_name"], $a_file["name"], "lm", "Modules/LearningModule");
+            $new_id = $imp->importObject(""/* TODO PHP8-REVIEW This must be NULl or an object */, $a_file["tmp_name"], $a_file["name"], "lm", "Modules/LearningModule"); //
```
Comment (mjansenDatabay) [inline Services/Help/classes/class.ilObjHelpSettings.php:80]: I would pass `null` here, but I am not sure.

**Comment by alex40724** [comment]
Thanks, I will check the Todos...

File: Services/Help/classes/class.ilObjHelpSettings.php (hunk 1 above)
Comment (alex40724) [reply Services/Help/classes/class.ilObjHelpSettings.php:80]: Yes, definitely. This parameter seems to be obsolete anyway, I will have to check the import component...

**Comment by alex40724** [comment]
Done.

**Pull request merged**